src/wazuh_db/schema_agents.sql

```diff
@@ -375,6 +375,10 @@ CREATE TABLE IF NOT EXISTS vuln_cves (
     version TEXT,
     architecture TEXT,
     cve TEXT,
+    detection_time TEXT DEFAULT '',
+    severity TEXT DEFAULT '',
```

Comment:
We can discuss with the team if this field needs a **CHECK** constraint or not.

Reply:
I added the CHECK constraints as we discussed.